### PR TITLE
fix for TaskPublisherGo.py

### PR DESCRIPTION
**Pull request by mapellidario**
Related to https://github.com/dmwm/CRABServer/issues/7379

I am testing this directly on prod, i am sorry for this

### CODE REVIEW

**Comment by mapellidario** [comment]
The test in production looks good:

```plaintext
2022-08-26 18:46:35,738:ERROR: migration terminally failed for /DYToLL_M-50_TuneCP5_14TeV-pythia8/Run3Summer21DRPremix-120X_mcRun3_2021_realistic_v6-v2/GEN-SIM-DIGI-RAW#073203cd-525c-4bbd-a2ae-923ee6339d1a

2022-08-26 18:46:35,738:ERROR: migration status details:
[{'create_by': 'service@crab-prod-tw01.cern.ch', 'creation_date': 1658961929, 'last_modification_date': 1658962357, 'last_modified_by': 'service@crab-prod-tw01.cern.ch', 'migration_input': '/DYToLL_M-50_TuneCP5_14TeV-pythia8/Run3Summer21DRPremix-120X_mcRun3_2021_realistic_v6-v2/AODSIM#00e43717-244e-438b-b2e9-81b3dec21175', 'migration_request_id': 3310703, 'migration_server': None, 'migration_status': 9, 'migration_url': 'https://cmsweb-prod.cern.ch:8443/dbs/prod/global/DBSReader', 'retry_count': 3}, {'create_by': 'service@crab-prod-tw01.cern.ch', 'creation_date': 1658961930, 'last_modification_date': 1658962357, 'last_modified_by': 'service@crab-prod-tw01.cern.ch', 'migration_input': '/DYToLL_M-50_TuneCP5_14TeV-pythia8/Run3Summer21DRPremix-120X_mcRun3_2021_realistic_v6-v2/GEN-SIM-DIGI-RAW#073203cd-525c-4bbd-a2ae-923ee6339d1a', 'migration_request_id': 3310705, 'migration_server': None, 'migration_status': 9, 'migration_url': 'https://cmsweb-prod.cern.ch:8443/dbs/prod/global/DBSReader', 'retry_count': 3}]
2022-08-26 18:46:35,738:ERROR: Migration failed for /DYToLL_M-50_TuneCP5_14TeV-pythia8/Run3Summer21DRPremix-120X_mcRun3_2021_realistic_v6-v2/GEN-SIM-DIGI-RAW#073203cd-525c-4bbd-a2ae-923ee6339d1a
```

**Comment by belforte** [comment]
thanks Dario !!!!!

File: src/python/Publisher/TaskPublishGo.py [hunk 1]
```diff
@@ -309,7 +314,7 @@ def checkBlockMigration(taskname, migrateApi, block, migLogDir):
         migCreation = datetime.fromtimestamp(creationDate).strftime('%Y-%m-%d,%H:%M:%S')
         # FiledMigFile format is CSV: creationDate, block, taskname
```
Comment (belforte) [inline src/python/Publisher/TaskPublishGo.py:315]: please update this comment as well

**Review by belforte — CHANGES_REQUESTED** [review]

File: src/python/Publisher/TaskPublishGo.py (hunk 1 above)
Comment (belforte) [reply src/python/Publisher/TaskPublishGo.py:315]: I mean Line 310. SOmehow the highlight is in the wrong line

Comment (mapellidario) [reply src/python/Publisher/TaskPublishGo.py]: Uh, I messed it, thank you!

Comment (belforte) [reply src/python/Publisher/TaskPublishGo.py]: +1

**Comment by belforte** [comment]
all good now, thanks

**Comment by mapellidario** [comment]
As agreed, I am merging this PR

**Pull request merged**